build-logic/src/main/kotlin/dokkabuild/DokkaBuildProperties.kt

```diff
@@ -48,11 +49,46 @@ abstract class DokkaBuildProperties @Inject constructor(
     val kotlinLanguageLevel: Provider<KotlinVersion> =
         dokkaProperty("kotlinLanguageLevel", KotlinVersion::fromVersion)
 
+    val integrationTestParallelism: Provider<Int> =
+        dokkaProperty("integration_test.parallelism", String::toInt)
+            // fallback & warn if the old property is used
+            .orElse(
+                providers.gradleProperty("dokka_integration_test_parallelism")
+                    .map {
+                        logger.warn("dokka_integration_test_parallelism is deprecated - use org.jetbrains.dokka.integration_test.parallelism instead")
+                        it.toInt()
+                    }
+            )
```

Comment:
A nice thing to have, although I don't think it's used much outside of our CI (GitHub/TC) and especially by external people. 

So if the use of this property is corrected in CI (you now have access to change TC configurations), and you can notify Dokka's maintainers in Slack that this property has been changed, I think there's no need to keep the compatibility like that. This idea applies to other internal things too :)

Reply:
Can do! I was concerned about forgetting to update these properties in whatever miscellaneous places (which could also be on on developers' machines too) after this chain of PRs is completed, but it's not such a big problem.